src/plugins/bubble/bubble.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * @title WET-BOEW Hello world plugin
+ * @overview Plugin contained to show an example of how to create your custom WET plugin
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp
+ */
+( function( $, window, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "wb-bubble",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+
+	/**
+	 * Initiate chat wizard bubble
+	 * @method initiateBubble
+	 * @param {jQuery DOM element} $selector Element which is the actual bubble
+	 */
+	initiateBubble = function( $selector ) {
+
+		var $footer = $( "#wb-info" );
+
+		// Hide basic form on load, show chat bubble instead
+		$selector.fadeIn( "slow" );
+
+		// Add some white space over the footer for the bubble to sit
+		$footer.addClass( componentName + "-mrgn" );
+
+		// Ensure that the bubble does not go passed the footer
+		if ( $footer.length ) {
+
+			// Keep the bubble sticky while scrolling Y until user reaches the footer
+			var stickyUntilFooter = function( $element ) {
+
+
+				// Equals to bubble default bottom value in CSS
+				var bottomY = 30;
+
+				if ( $( window ).scrollTop() >= $( document ).outerHeight() - $( window ).outerHeight() - $footer.outerHeight() ) {
+					$element.css( {
+						bottom: ( $footer.outerHeight() - ( $( document ).outerHeight() - $( window ).outerHeight() - $( window ).scrollTop() ) + bottomY )
+					} );
+
+					//console.log($element);
+				} else {
+					$element.css( {
+						bottom: bottomY
+					} );
+				}
+			};
+
+			// Correct bubble positionning on load, on resize an on Y scroll if necessary
+			stickyUntilFooter( $selector );
+
+			$( window ).on( "resize scroll", function() {
+				stickyUntilFooter( $selector );
+			} );
+		} else {
+
+			$( "main" ).after( "<footer id=\"wb-info\"></footer>" );
+			$( "#wb-info" ).addClass( componentName + "-mrgn" );
+
+		}
+
+		// Close notification aside bubble
+		$( ".notif-close", $selector ).on( "click", function( event ) {
+			event.preventDefault();
+			$( this ).parent().hide();
+			$selector.focus();
+
+			// Do not show notification on next load
+			localStorage.setItem( "wb-chtwzrd-notif", 1 );
```

Comment:
Rename this for the componentName + "notif" instead of "wb-chtwzrd". I think the intend is to be in the sessionStorage, not in the localStorage

Also, when this is read? It is set but never used. @GormFrank what is the intend? Is it the same situation with the current chat wizard? If so we might need to do some cleanup.

Reply:
@ctopCanada as discussed, please use the session storage and take it in consideration when the plugin initiate.